lib/new_relic/agent/span_event_primitive.rb

```diff
@@ -29,12 +29,16 @@ module SpanEventPrimitive
       CATEGORY_KEY = 'category'
       HTTP_URL_KEY = 'http.url'
       HTTP_METHOD_KEY = 'http.method'
+      HTTP_REQUEST_METHOD_KEY = 'http.request.method'
```

Comment:
Curious, what could the difference be between `HTTP_METHOD_KEY` and `HTTP_REQUEST_METHOD_KEY`.

Reply:
The attribute names differ but their values are identical. There is a desire to rename `http.method` to `https.request.method`. We're not ready to introduce any backwards compatibility issues by renaming it at this time, so instead we are simply adding the new name. It's possible that we'll now proceed to deprecate the old name and stop using it at a future date.

Reply:
Makes sense ✨